packages/orbit-components/src/Loading/README.md

```diff
@@ -16,15 +16,17 @@ After adding import into your project you can use it simply like:
-| loading    | `boolean`       | `false` | If `true`, the Loading will be displayed. Loading which doesn't have a children is always shown, even if `loading` prop is set to `false`. |
-| **type**   | [`enum`](#enum) |         | The type of the Loading.                                                                                                                   |
-| text       | `Translation`   |         | The text of the Loading.                                                                                                                   |
-| customSize | `number`        |         | Allows you to define custom size for circle loader                                                                                         |
+| Name        | Type                      | Default | Description                                                                                                                                |
+| :---------- | :------------------------ | :------ | :----------------------------------------------------------------------------------------------------------------------------------------- |
+| children    | `React.Node`              |         | The content of the Loading. [See Functional specs](#functional-specs)                                                                      |
+| dataTest    | `string`                  |         | Optional prop for testing purposes.                                                                                                        |
+| id          | `string`                  |         | Set `id` for `Loading`                                                                                                                     |
+| loading     | `boolean`                 | `false` | If `true`, the Loading will be displayed. Loading which doesn't have a children is always shown, even if `loading` prop is set to `false`. |
+| **type**    | [`enum`](#enum)           |         | The type of the Loading.                                                                                                                   |
+| text        | `Translation`             |         | The text of the Loading.                                                                                                                   |
+| customSize  | `number`                  |         | Allows you to define custom size for circle loader                                                                                         |
+| asComponent | `string \| React.Element` | `div`   | The component used for the root node.                                                                                                      |
+| title       | `string`                  |         | Optional prop for image title.                                                                                                             |
```

Comment:
The description can be improved. Or at least refer to the accessibility documentation